MSAL/test/unit/native_auth/public/error/SignUpStartErrorTests.swift

```diff
@@ -30,7 +30,7 @@ final class SignUpStartErrorTests: XCTestCase {
     private var sut: SignUpStartError!
 
     func test_totalCases() {
-        XCTAssertEqual(SignUpStartError.ErrorType.allCases.count, 4)
+        XCTAssertEqual(SignUpStartError.ErrorType.allCases.count, 5)
```

Comment:
This test was created to remind us that if we add new errors, we should add a test for those errors 🙂.

The test `test_defaultErrorDescription` should be updated with the new error as well.

Reply:
Thanks Diego, updated ;)